codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/auth/AuthParametersResolverGenerator.java

```diff
@@ -64,7 +64,11 @@ public GoWriter.Writable generate() {
     private GoWriter.Writable generateResolvers() {
         return (writer) -> {
             for (var resolver: resolvers) {
-                writer.write("$T(ctx, params, input, options)", resolver.resolver());
+                if (resolver.resolver().getName().equals("bindAuthEndpointParams")) {
+                    writer.write("err = $T(ctx, params, input, options)", resolver.resolver());
```

Comment:
I'm not exactly fond of the way is this written. We shouldn't need to branch if you change it so _all_ auth param resolvers by design return errors. As written this could have issues in the future if we add additional bindings that return errors.

I'm not going to block on the review though so either fixandship or leave it.

Reply:
I got what you mean, I can change all resolvers from go v2 side to return error considering we only have 2 resolvers for now and that could hide some resolver info